tests/security/cc/cc_audit_test_setup.pm

```diff
@@ -41,6 +41,9 @@ sub run {
     # Install audit packages
     zypper_call('in audit audit-audispd-plugins');
 
+    # Install tool packages
+    zypper_call('in wget');
```

Comment:
wget will be used in the other test case ?

Reply:
or it is necessary for the next testing.

Reply:
Yes, we need wget the audit-test code to run.